core/logger/zap.go

```diff
@@ -116,7 +33,7 @@ func makeEncoderConfig(cfg Config) zapcore.EncoderConfig {
 }
 
 func (l *zapLogger) SetLogLevel(lvl zapcore.Level) {
-	l.config.Level.SetLevel(lvl)
+	l.level.SetLevel(lvl)
```

Comment:
Are we going to eliminate this part of runtime reconfiguration? If so, we could simplify more here.

Reply:
I believe the uses of it has already been removed, no?